### DISCUSSION TITLE
Log Level Verbose

### DISCUSSION THREAD

**Discussion by erayymz** [Flexget/Flexget#3671] [Q&A]
How do I set the log level so that it logs VERBOSE into to log file?

**Comment by BrutuZ** [comment]
`--loglevel VERBOSE` https://flexget.com/CLI#flexget-level-arguments

**Comment by erayymz** [comment]
How do I do this for the daemon? Currently, I am running it in a startup using crontab. I simply have the following:

@reboot /usr/local/bin/flexget daemon start -d

Should I just modify that to following:

@reboot /usr/local/bin/flexget --loglevel VERBOSE daemon start -d

**Reply by BrutuZ** [reply]
Correct
`verbose` might need to be in lower caps instead, I don't know if the command parser handles that (probably does, but heads-up anyways)

**Reply by erayymz** [reply]
Following worked great. Thx.

@reboot ~/flexget/bin/flexget --logfile flexgetd.log --loglevel verbose daemon start -d
